frontend/cypress/plugins/index.js

```diff
@@ -19,6 +19,14 @@ module.exports = (on, config) => {
   // `on` is used to hook into various events Cypress emits
   // `config` is the resolved Cypress config
 
+  // enable printing to terminal
+  on("task", {
+    log(message) {
+      console.log(message)
```

Comment:
Don't have anything against this but doesn't console.log already work? The annoying part is you'll just have to open the console in your test

Reply:
Based on my understanding, which is based on the internet, which is never wrong, I *believe* Cypress tasks let you register bits of JS code that run in the Node instance, i.e. the terminal, when `cy.task` is called.

Whatever's actually happening under the hood, all I'm aware of is that `cy.task("log", "whatever")` makes `whatever` show up in my terminal 😛

Reply:
Ahh I clearly missed the comment. Thought this was for logging into the browser, not terminal